db_classes.py

```diff
@@ -87,21 +87,21 @@ class DependencyType(Document):
 
     # task_type # a dependency type groups the dependencies of 1 task type
     # dependency # a dependency type groups 0+ dependencies
-    allowed_values = LazyReferenceField('AllowedValue', required=True) # a dependency type's manner (subtask / blocker) is specified by 1 allowed value
+    allowed_values = LazyReferenceField('AllowedValue', required=True, passthrough=True) # a dependency type's manner (subtask / blocker) is specified by 1 allowed value
 
 
 class Dependency(Document):
     # no intrinsic attributes
 
-    depended_on_task = LazyReferenceField('Task', required=True) # a dependency expresses a task's dependence on 1 task
-    dependent_task = LazyReferenceField('Task', required=True) # a dependency expresses the dependence of 1 task
-    dependency_type = LazyReferenceField('DependencyType', required=True) # a dependency is of 1 dependency type
+    depended_on_task = LazyReferenceField('Task', required=True, passthrough=True) # a dependency expresses a task's dependence on 1 task
+    dependent_task = LazyReferenceField('Task', required=True, passthrough=True) # a dependency expresses the dependence of 1 task
+    dependency_type = LazyReferenceField('DependencyType', required=True, passthrough=True) # a dependency is of 1 dependency type
 
 
 class FilteredView(Document):
     name = StringField(required=True)
     query = StringField(required=True)
 
-    owner_user = LazyReferenceField('User') # a filtered view is owned by 1 user
-    shared_with_users = ListField(LazyReferenceField('User')) # a filtered view is shared with 0+ (non-owning) users
-    workspaces = ListField(LazyReferenceField('Workspace', required=True)) # a filtered view belongs to 1+ workspaces
+    owner_user = LazyReferenceField('User', passthrough=True) # a filtered view is owned by 1 user
+    shared_with_users = ListField(LazyReferenceField('User', passthrough=True)) # a filtered view is shared with 0+ (non-owning) users
+    workspaces = ListField(LazyReferenceField('Workspace', required=True, passthrough=True)) # a filtered view belongs to 1+ workspaces
```

Comment:
tiny thing but for the record the little minus that github displays here is indicating a whitespace error that occurs when the final line ending in a file gets deleted